### DISCUSSION TITLE
Allow searching for repositories based on number of contributors?

### DISCUSSION THREAD

**Discussion by sideshowbarker** [community/community#12969] [Repositories]
I’d like to propose that GitHub’s repository-search feature be enhanced to allow searching for repositories based on number of contributors.

As documented at https://docs.github.com/en/search-github/searching-on-github/searching-for-repositories, the search feature already allows searching for repositories based on:
- number of followers
- number of forks
- number of stars

Given that, adding the ability to search for repositories based on the number of contributors seems like a logical/natural enhancement to the existing set of repository-search capabilities.

**Comment by JustinGOSSES** [comment]
Would like this feature as part of searching for repositories within organizations that might be more welcoming to new contributors
